config/default/kustomization.yaml

```
@@ -1,9 +1,8 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
-# Adds namespace to all resources. Keep it in capm3-system, as it is a
-# dependency for CAPM3
-namespace: capm3-system
+# Adds namespace to all resources
+namespace: ipam-provider-m3-system
 
 namePrefix: ipam-
```

This prefix must match so that it + the namespace `name` (=system) becomes the `namespace` above.
```suggestion
namePrefix: ipam-metal3-
```